standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -278,6 +278,7 @@ private void startLocalSegmentStore(int segmentStoreId) throws Exception {
                         .with(ServiceConfig.KEY_FILE, this.keyFile)
                         .with(ServiceConfig.CERT_FILE, this.certFile)
                         .with(ServiceConfig.ENABLE_TLS_RELOAD, this.enableTlsReload)
+                        .with(ServiceConfig.LISTENING_IP_ADDRESS, this.enableTls ? "localhost" : "")
```

Comment:
Curious, shouldn't this be `PUBLISHED_IP_ADDRESS` instead? 

I understand that you are trying to change this from IP address `127.0.1.1` that the Segment store listens on to `localhost`, so that hostname verification can be turned on for client/controller-to-segment store communication.

Reply:
Thanks Ravi, yes, your understanding is correct, this is for hostname verification for segmentstore. 

Actually I want to ask what's the difference between Published and Listening address? The reason I use Listening address is from this [comment](https://github.com/pravega/pravega/blob/c7ac009970787df2633163644691aa3d72fc1a4e/segmentstore/server/src/test/java/io/pravega/segmentstore/server/store/ServiceConfigTests.java#L28), looks like if we don't specify published address, it will default to listening address.

Reply:
Yes, if published IP and port are not registered with the Controller for the Segment Store service, the Controller returns the service's listening IP and port to the clients afaik. cc @andreipaduroiu